js/app.js

```diff
@@ -0,0 +1,288 @@
+  { "title" : "Goldfinger", "year" : 1964, "actor" : "Sean Connery", "gross" : "$912,257,512" },
+  { "title" : "Live and Let Die", "year" : 1973, "actor" : "Roger Moore", "gross" : "$825,110,761" },
+  { "title" : "You Only Live Twice", "year" : 1967, "actor" : "Sean Connery", "gross" : "$756,544,419" },
+  { "title" : "The Spy Who Loved Me", "year" : 1977, "actor" : "Roger Moore", "gross" : "$692,713,752" },
+  { "title" : "Casino Royale", "year" : 2006, "actor" : "Daniel Craig", "gross" : "$669,789,482" },
+  { "title" : "Moonraker", "year" : 1979, "actor" : "Roger Moore", "gross" : "$655,872,400" },
+  { "title" : "Diamonds Are Forever", "year" : 1971, "actor" : "Sean Connery", "gross" : "$648,514,469" },
+  { "title" : "Quantum of Solace", "year" : 2008, "actor" : "Daniel Craig", "gross" : "$622,246,378" },
+  { "title" : "From Russia with Love", "year" : 1963, "actor" : "Sean Connery", "gross" : "$576,277,964" },
+  { "title" : "Die Another Day", "year" : 2002, "actor" : "Pierce Brosnan", "gross" : "$543,639,638" },
+  { "title" : "Goldeneye", "year" : 1995, "actor" : "Pierce Brosnan", "gross" : "$529,548,711" },
+  { "title" : "On Her Majesty's Secret Service", "year" : 1969, "actor" : "George Lazenby", "gross" : "$505,899,782" },
+  { "title" : "The World is Not Enough", "year" : 1999, "actor" : "Pierce Brosnan", "gross" : "$491,617,153" },
+  { "title" : "For Your Eyes Only", "year" : 1981, "actor" : "Roger Moore", "gross" : "$486,468,881" },
+  { "title" : "Tomorrow Never Dies", "year" : 1997, "actor" : "Pierce Brosnan", "gross" : "$478,946,402" },
+  { "title" : "The Man with the Golden Gun", "year" : 1974, "actor" : "Roger Moore", "gross" : "$448,249,281" },
+  { "title" : "Dr. No", "year" : 1962, "actor" : "Sean Connery", "gross" : "$440,759,072" },
+  { "title" : "Octopussy", "year" : 1983, "actor" : "Roger Moore", "gross" : "$426,244,352" },
+  { "title" : "The Living Daylights", "year" : 1987, "actor" : "Timothy Dalton", "gross" : "$381,088,866" },
+  { "title" : "A View to a Kill", "year" : 1985, "actor" : "Roger Moore", "gross" : "$321,172,633" },
+  { "title" : "License to Kill", "year" : 1989, "actor" : "Timothy Dalton", "gross" : "$285,157,191" }
+];
+
+let bondTitles = [];
+
+for (let i = 0; i < bondFilms.length; i++) {
+	bondTitles.push(bondFilms[i].title);
+}
+
+console.log(bondTitles);
+
+let oddBonds = [];
+
+for (let i = 0; i < bondFilms.length; i++) {
+	if (bondFilms[i].year % 2 === 1) {
+		oddBonds.push(bondFilms[i].year);
+	}
+}
+
+console.log(oddBonds);
+
+let totalGross = 0;
+
+for (let i = 0; i < bondFilms.length; i++) {
+	//loops through the bondFilms property "gross" and replaces all non-alphanumeric characters with empty spaces(\W selects all non-alphnumeric and g allows for it to continue after it finds the first instance)
+	let cleanGross = bondFilms[i].gross.replace(/\W/g, '');
+	//converts the cleanGross string to a number in base 10
+	let convertGross = parseInt(cleanGross, 10);
```

Comment:
I'd leave out the 10 because it's the default for an optional param and it suggests that this data was previously binary/octal/hexadecimal/etc, which it wasn't